host/fs/fs.go

```diff
@@ -58,6 +62,12 @@ type File struct {
 
 // Ioctl sends an ioctl to the file handle.
 func (f *File) Ioctl(op uint, data uintptr) error {
+	if isMIPS {
```

Comment:
I'd remove `ioctl_mips_like.go` and `ioctl_other.go` and replace this line with `if runtime.GOARCH == "mips" || runtime.GOARCH == "mipsle" {`. It costs the same as it will be evaluated at compile time.

Reply:
That's true but I prefer this pattern because there can't be typo (like the one I had done with "mipsel") and I tend to break stuff all the time otherwise; i.e. I don't trust myself.